src/network/webtask.cpp

```diff
@@ -179,6 +180,25 @@ void WebTask::slotStart(int timeoutMillis) {
         return;
     }
 
+    // This block is intended to fix the bug:
+    // https://bugs.launchpad.net/mixxx/+bug/1983206
+    // Some of the tracks returns (404-Empty XML)
+    // in MusicBrainz Task. This block simply
+    // saves the looping state before
+    // it resets. To skip empty responses
+    // without affecting the other states-tasks
+    // this saving was needed.
+
+    // m_taskCounter is set to 0 because, if
```

Comment:
We need to either explicit tell the WebTask to ignore 404 or filter it out via virtal override function.
Using a counter and derive unrelated decisions from it is hard to understand later.

Reply:
I simply did ignore 404 but then the whole tasks failed. Such as AcoustID was not returning the not found message to the user. 

So filtering it in MBTask would be better in this case.